framework/packages/abstract-core/src/native/ibc_host.rs

```diff
@@ -130,6 +134,8 @@ pub enum ExecuteMsg {
 #[derive(QueryResponses)]
 #[cfg_attr(feature = "interface", derive(cw_orch::QueryFns))]
 pub enum QueryMsg {
+    #[returns(cw_ownable::Ownership<String> )]
```

Comment:
Same as other PR, I think this returns Ownership<Addr>

Reply:
I might be wrong on this lol! It looks like the cw-ownable macro returns Ownership<String>...

Reply:
> I might be wrong on this lol! It looks like the cw-ownable macro returns Ownership...

I think markdown ate your generic on `Ownership<String>`